apps/meta-data-utils/src/tableQuery.ts

```diff
@@ -1,10 +1,15 @@
-import {
+import type {
```

Comment:
ah, is this the way to import typescript types?

Reply:
this is the new 'prefered' vite way to import types, think it give the compiler more hints , i also thinks is clearer for the developer